templates/environment/versions/cf-v1.3.1.yml

```diff
@@ -0,0 +1,371 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+Description: CloudFormation environment template for infrastructure shared among Copilot workloads.
+Metadata:
+  Version: 'v1.3.1'
```

Comment:
There is no difference in this template besides the version here, we had to create a new version so that the Lambda runtime version updates can be picked up by `env upgrade`